src/liballoc/slice.rs

```diff
@@ -1722,6 +1722,71 @@ impl<T> [T] {
         // NB see hack module in this file
         hack::into_vec(self)
     }
+
+    /// Creates a vector by repeating a slice `n` times.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// assert_eq!([1, 2].repeat(3), vec![1, 2, 1, 2, 1, 2]);
+    /// ```
+    #[stable(feature = "repeat_slice", since = "1.27.0")]
```

Comment:
All new APIs should start off as unstable. Just invent a new feature name for it, like "repeat_generic_slice" or so.

Reply:
Ok!